tests/test_dummy.py

```diff
@@ -30,7 +33,8 @@ def test_portrait():
     device = dummy(rotate=1)
 
     # Use the same drawing primitives as the demo
-    with canvas(device) as draw:
+    # Enable dither for extra test coverage
+    with canvas(device, dither=True) as draw:
```

Comment:
Would be better to create a separate test for dither - create a `mode="1"` dummy device, and draw some coloured boxes onto a dithered canvas, and check that resultant image is dithered. Whether pillow's dither implementation works consistently/randomly may thwart this though